chwifi

```diff
@@ -138,13 +138,8 @@ display_daily_password() {
 }
 
 connect_dispatch() {
-    # If home, then connect and update cached passwords when connection is given
-    if [[ "$1" =~ home ]]; then
-        printf '%s\n' "Connecting to home"
-        connection_routine "$network_manager_home_profile"
-        
     # If work, then find password, connect and update cached passwords when connection is given
-    elif [[ "$1" =~ work ]]; then
+    if [[ "$1" =~ work ]]; then
```

Comment:
```suggestion
    if [[ "$1" =~ ^$work_profile$ ]]; then
```

Shouldn't this be the configured to use `$work-profile`

Reply:
`$network_manager_work_profile`, actually, but yes. 861499d fixes this.